snmptrap.php

```diff
@@ -29,4 +29,20 @@
     $entry[] = $f;
 }
 
-snmptrap($entry);
+$hostname = trim($entry[0]);
+$ip = str_replace(array("UDP:","[","]"), "", $entry[1]);
+$ip = trim(strstr($ip, ":", true));
+
+$device = @dbFetchRow('SELECT * FROM devices WHERE `hostname` = ?', [$hostname]);
```

Comment:
You should check the ip field too.  This might work.
```php
$device = dbFetchRow('SELECT * FROM devices WHERE `hostname`=? OR `hostname`=? OR `ip`=?', [$hostname, $ip, inet_pton($ip)]);
```

Remove the @ from these queries